assets/src/dashboard/components/pageStructure/navigationComponents.js

```diff
@@ -43,72 +42,76 @@ export const Header = styled.h1`
   }
 `;
 
-export const NavButton = styled(Button)`
-  margin-bottom: 0;
-  margin-top: 0;
-  background-color: ${({ theme }) =>
-    theme.DEPRECATED_THEME.colors.foreground.gray12};
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.black};
-  border-radius: 4px;
-  border-color: ${({ theme }) =>
-    theme.DEPRECATED_THEME.colors.foreground.gray12};
-  height: 36px;
-
-  &:hover,
-  &:focus,
-  &:focus-within {
-    background-color: ${({ theme }) =>
-      theme.DEPRECATED_THEME.colors.foreground.gray16};
-    color: ${({ theme }) => theme.DEPRECATED_THEME.colors.black};
-  }
-`;
+export const NewStoryButton = styled(Button)(
+  ({ theme }) => css`
+    margin-bottom: 0;
+    margin-top: 0;
+    border: 2px solid ${theme.colors.border.defaultActive};
+    border-radius: ${theme.borders.radius.medium};
+    transition: border-color 0.25s linear, background-color 0.25s linear,
+      color 0.25s linear;
+
+    :focus {
+      box-shadow: none;
+    }
+
+    :hover {
+      color: ${theme.colors.bg.primary};
+      background-color: ${theme.colors.interactiveBg.brandPress};
+      border-color: ${theme.colors.interactiveFg.brandPress};
+    }
+  `
+);
 
 export const NavList = styled.ul`
   margin: 0;
   padding: 0;
 `;
 
 export const NavListItem = styled.li`
```

Comment:
The spacing between items in this list seem a little more compact than in the Figma designs by 4ish pixels both in the top lists and the Settings/Support list at the bottom.

Reply:
I'll fix that!